drake/util/timeUtil.h

```diff
@@ -0,0 +1,40 @@
+#ifndef DRAKE_WINTIME_H
+#define DRAKE_WINTIME_H
```

Comment:
sounds great.  didn't realize it was there.  (although the `#include<crono>` is already failing on travis)